CMSIS/Core/Include/core_cm33.h

```diff
@@ -519,7 +519,7 @@ typedef struct
   __IOM uint32_t AFSR;                   /*!< Offset: 0x03C (R/W)  Auxiliary Fault Status Register */
   __IM  uint32_t ID_PFR[2U];             /*!< Offset: 0x040 (R/ )  Processor Feature Register */
   __IM  uint32_t ID_DFR;                 /*!< Offset: 0x048 (R/ )  Debug Feature Register */
-  __IM  uint32_t ID_ADR;                 /*!< Offset: 0x04C (R/ )  Auxiliary Feature Register */
+  __IM  uint32_t ID_AFR;                 /*!< Offset: 0x04C (R/ )  Auxiliary Feature Register */
```

Comment:
Changing exiting struct members is a breaking change. We cannot easily do this.

Reply:
Please revert this change.

Reply:
Reverted.

Reply:
Are these changes going to be merged in this PR? If not I can move them to a new PR (with anon union 😉 and appropriate pragmas for compiler support).